bundles/io/org.eclipse.smarthome.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttService.java

```diff
@@ -34,6 +41,8 @@
  * @author Davy Vanherbergen
  * @author Markus Rathgeb - Synchronize access to broker connections
  */
+@Component(immediate = true, service = { MqttService.class, ManagedService.class }, configurationPid = {
+        "org.eclipse.smarthome.mqtt" }, name = "MQTTConnectionService", property = "service.pid=org.eclipse.smarthome.mqtt")
 public class MqttService implements ManagedService {
```

Comment:
No, they are outdated. We only use configurations for DS components throughout ESH.